modelzoo/unet/nnunet.patch

```diff
@@ -29,7 +29,7 @@ diff -urN nnUNet/.github/ISSUE_TEMPLATE/all-issues.md nnUNetNew/.github/ISSUE_TE
 -
 -
 diff -urN nnUNet/.gitignore nnUNetNew/.gitignore
---- nnUNet/.gitignore	2023-02-09 08:27:01.510133303 +0000
+--- nnUNet/.gitignore	2023-02-18 06:51:01.252097744 +0000
 +++ nnUNetNew/.gitignore	1970-01-01 00:00:00.000000000 +0000
```

Comment:
@zhouyu5 , can we remove the change to .git and .gitignore? Or it will always change

Reply:
@zhouyu5 , @XinyaoWa , update my request, can we remove unnecessary changes of time in this patch?

Reply:
Already replace the diff patch with git patch. Now no diff on timestamp and .git file.